ci-operator/step-registry/openshift/e2e/test/qe/regenerate-cluster-cert/openshift-e2e-test-qe-regenerate-cluster-cert-commands.sh

```diff
@@ -154,31 +162,30 @@ oc adm restart-kubelet nodes --all --directive=RemoveKubeletKubeconfig
 # delete all pods to ensure that new kube-apiserver ca bundles are used by all pods
 oc adm reboot-machine-config-pool mcp/worker mcp/master
 oc adm wait-for-node-reboot nodes --all
-
 # now that trust is distributed, create new certificates using new roots
 oc adm ocp-certificates regenerate-leaf -n openshift-kube-apiserver-operator secrets node-system-admin-client
-oc adm ocp-certificates regenerate-leaf -n openshift-kube-apiserver secrets check-endpoints-client-cert-key control-plane-node-admin-client-cert-key  external-loadbalancer-serving-certkey internal-loadbalancer-serving-certkey kubelet-client localhost-recovery-serving-certkey localhost-serving-cert-certkey service-network-serving-certkey
+oc adm ocp-certificates regenerate-leaf -n openshift-kube-apiserver secrets check-endpoints-client-cert-key control-plane-node-admin-client-cert-key external-loadbalancer-serving-certkey internal-loadbalancer-serving-certkey kubelet-client localhost-recovery-serving-certkey localhost-serving-cert-certkey service-network-serving-certkey
 oc adm wait-for-stable-cluster
-
 # create new admin.kubeconfig
-oc config new-admin-kubeconfig > "${SHARED_DIR}/admin.kubeconfig"
+oc config new-admin-kubeconfig >"${SHARED_DIR}/admin.kubeconfig"
```

Comment:
IMO, this kind of changes should be reverted, as the former one are easier to read.